MaxText/layers/quantizations.py

```diff
@@ -136,13 +136,13 @@ def dot_general_cls(self, mesh_axes: Tuple[str, ...] = ()):
     )
     return aqt_dg_cls
 
-  def einsum(self, mesh_axes: Tuple[str, ...] = ()):
+  def einsum(self, mesh_axes: Tuple[str, ...] = (),):
     """Returns einsum configured with aqt params."""
     rhs_axis_metadata_wrapper = self._get_rhs_axis_metadata_wrapper(mesh_axes)
     aqt_einsum = functools.partial(
         aqt_flax.AqtEinsum(
             cfg=self.quant_dg,
-            lhs_quant_mode=self.quant_mode,
+            rhs_quant_mode=self.quant_mode,
```

Comment:
Do we need this change? I thought we just replace line related `jnp.einsum`?

Reply:
From the code, we are passing weights as rhs, which are quantized. Can we do training run to confirm its not broken?

Reply:
talked to Zhiyu offline, looks like this was a bug as well. Can we do anything else to check?

Reply:
As long as end-to-end training works should be fine. Have you double checked?